documentation/API-design-guidelines.md

```diff
@@ -1976,40 +1976,46 @@ response:
+  Whilst it might be considered harmless to proceed if both identify the same User, returning an error only when the two do not match would allow the API consumer to confirm the identity of the User associated with the access token, which they might otherwise not know. Although this functionality is supported by some APIs (e.g. Number Verification, KYC Match), for others it may exceed the scope consented to by the User.
 
-This specification defines the `device` object field as optional in API requests, specifically in cases where the API is accessed using a 3-legged access token, and the device can be uniquely identified by the token. This approach simplifies API usage for API consumers by relying on the device information associated with the access token used to invoke the API.
+  In this case, a `422 UNNECESSARY_IDENTIFIER` error code MUST be returned unless the scope of the API allows it to explicitly confirm whether or not the supplied identity matches that bound to the Three-Legged Access Token.
 
-## Handling of device information:
+If an API provider issues Two-Legged Access Tokens for use with the API, the following error may occur:
+- **Neither a Three-legged Access Token nor an explicit User identifier (device or phone number) are provided by the API consumer.**
 
-### Optional device object for 3-legged tokens:
+  One or other MUST be provided to identify the User.
 
-- When using a 3-legged access token, the device associated with the access token must be considered as the device for the API request. This means that the device object is not required in the request, and if included it must identify the same device, therefore **it is recommended NOT to include it in these scenarios** to simplify the API usage and avoid additional validations.
+  In this case, a `422 MISSING_IDENTIFIER` error code MUST be returned, indicating that the API provider cannot identify the User from the provided information.
 
-### Validation mechanism:
+The documentation template below is RECOMMENDED to be used as part of the `info.description` API documentation to explain to the API consumer how the pattern works.
 
-- The server will extract the device identification from the access token, if available.
-- If the API request additionally includes a `device` object when using a 3-legged access token, the API will validate that the device identifier provided matches the one associated with the access token.
-- If there is a mismatch, the API will respond with a 403 - INVALID_TOKEN_CONTEXT error, indicating that the device information in the request does not match the token.
+This template is applicable to CAMARA APIs which:
+- require the User (i.e. Resource Owner) to be identified; and
+- may have implementations which accept Two-Legged Access Tokens; and
+- do NOT allow the API to confirm whether or not the optional User identifier (`device` or `phoneNumber`) matches that associated with the Three-Legged Access Token
```

Comment:
Would be great if we could stick to the same term as used below  "subject identifier"

Reply:
Unfortunately "subject identifier" is not a defined CAMARA term, and so potentially ambiguous in normative sections of a standard. I am trying to use terms from the ICM glossary in normative sections (e.g. "User"), but felt "subject identifier" was a more natural term for the informative `info.description`.

I wouldn't expect application developers to necessarily read our glossary, but API developers should be aware of it.